supernode/services/senseregister/task.go

```diff
@@ -537,18 +537,19 @@ func (task *Task) ValidateAndRegister(ctx context.Context, ticket []byte, creato
 						return nil
 					}
 
-					// confirmations := task.waitConfirmation(ctx, nftRegTxid, 10, 30*time.Second, 55)
-					// err = <-confirmations
-					// if err != nil {
-					// 	return errors.Errorf("wait for confirmation of reg-art ticket %w", err)
-					// }
-
-					// TODO: update in next step
-					// if err = task.storeIDFiles(ctx); err != nil {
-					// 	log.WithContext(ctx).WithError(err).Errorf("store id files")
-					// 	err = errors.Errorf("store id files: %w", err)
-					// 	return nil
-					// }
+					// Wait for action ticket to be activated by walletnode
+					confirmations := task.waitActionActivation(ctx, nftRegTxid, 10, 30*time.Second)
+					err = <-confirmations
+					if err != nil {
+						return errors.Errorf("wait for confirmation of reg-art ticket %w", err)
+					}
+
+					// Store dd_and_fingerprints into Kademlia
+					if err = task.storeIDFiles(ctx); err != nil {
```

Comment:
this should be implemented at finalization of step 10 - after primary SN got activation ticket and verified it

Reply:
done.